frontend/src/streamlist/StreamsTablePopulator.jsx

```diff
@@ -1,9 +1,8 @@
 import React from "react";
-import { Box, Container } from "@material-ui/core";
 import PropTypes from "prop-types";
-import StreamsTable from "./StreamsTable";
+import ActiveStreamsTable from "./StreamsTable";
```

Comment:
if there's only a single export can you forgo using `as` to change the names of stuff you import?
also could you clarify why we're renaming to `ActiveStreamsTable` ?

Reply:
typo. Removed in 965f203